### DISCUSSION TITLE
How do I remove the comments and minify the generated CSS output from Tailwind?

### DISCUSSION THREAD

**Discussion by arentsen** [tailwindlabs/tailwindcss#2160] [Help] [answered]
My Craft project is in development stage and if I build my tailwind CSS file, it is huge because of all the comments in it. I would just like to minify it (even without using purgeCSS for all the unused classes at this point) but I cannot find any information on it. How do I do this?

**Comment by simonswiss** [comment, marked as answer]
Hi @arentsen! 👋

For this you could use [cssnano](https://cssnano.co/), another PostCSS plugin.

You can add the plugin to your your array in your `postcss.config.js` like so:

```javascript
module.exports = {
  plugins: [
    // ...
    require('tailwindcss'),
    require('autoprefixer'),
    require('cssnano'),
    // ...
  ]
}
```

It might be worth only running `cssnano` for production:

```javascript
module.exports = {
  plugins: [
    require('tailwindcss'),
    require('autoprefixer'),
    ...process.env.NODE_ENV === 'production'
      ? [require('cssnano')]
      : []
  ]
}
```
Hope it helps! 👍

**Reply by arentsen** [reply]
Helps very much!! thanks
